build.sh

```diff
@@ -235,37 +247,72 @@ crontab -l
 EOF
 
 
-ssh $osname  "$VM_SHUTDOWN_CMD"
+if [ -e "hooks/sysUpdate.sh" ]; then
+  echo "hooks/sysUpdate.sh"
+  cat "hooks/sysUpdate.sh"
+  ssh $osname sh<"hooks/sysUpdate.sh"
 
-sleep 30
+  # We need to reboot after system updates
+  restart_and_wait
+fi
```

Comment:
if there is no "sysUpdate.sh",  we need to shutdown_and_wait  here, to keep the same logic as before.

Reply:
maybe, we also need restart_and_wait  no matter there is a sysUpdate.sh or not.

Reply:
shutdown_and_wait is indeed called no matter what on line 274 ... it just adds a restart_and_wait in the case that there _is_ a sysUpdate.sh, since after updates you may need to restart and then perform the actions of postBuild.sh.  So this does keep the same prior behavior if there is no sysUpdate.sh

Reply:
ok, with your reorg request, it does make sense to have a restart_and_wait after postBuild.sh ...